java/build.gradle

```diff
@@ -74,6 +74,16 @@ task javadocJar(type: Jar) {
     from javadoc
 }
 
+distZip {
+    archiveFileName = "${project.name}.zip"
+    duplicatesStrategy = DuplicatesStrategy.EXCLUDE
+}
+
+distTar {
+    archiveFileName = "${project.name}.tar"
+    duplicatesStrategy = DuplicatesStrategy.EXCLUDE
+}
+
```

Comment:
I also revert https://github.com/scalar-labs/scalar-admin/commit/945d9325b693d886f57cd0305583060fc097ad08 since we need to set up the distTar task for the docker task.